frameworks/Rust/actix/actix-core.dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM rust:1.35
+FROM rust:latest
```

Comment:
Can you lock all these down to a particular version?

It is important that each run be reproducible based on the git commit hash, and if you use `rust:latest`, we will not be able to guarantee the version used on any particular run.

Reply:
@msmith-techempower you got it; done in latest commit!